backup_monkey/core.py

```diff
@@ -22,7 +21,11 @@
 __all__ = ('BackupMonkey', 'Logging')
 log = logging.getLogger(__name__)
 
+from splunk_logging import SplunkLogging  
+from status import BackupMonkeyStatus as s
```

Comment:
``s`` is not descriptive enough - especially since the context is the whole file.